setup.py

```diff
@@ -24,7 +23,6 @@
 
 setup(
     name="breathe",
-    version=__version__,
```

Comment:
@2bndy5 As far as I know this determines the version seen on PyPI with how the packaging scripts runs, but I am no way an expert on this. Are you sure removing this line won't cause issues?

I think you can verify by tagging a fake version locally (a to-be 4.32.0) and then running `./mkrelease pack` from repo root. That'll dump what is normally uploaded in `mkrelease_tmp/dist/*.wlh`. Inside that there should be some metadata file iirc.

Reply:
I just read https://github.com/michaeljones/breathe/issues/714#issuecomment-974226777, so presumably this does work rightly with current version.

After this review round I think it's time to just test this and if it works it works.

Reply:
This line has nothing to do with pypi. It was only ever a convenience for users to programmatically get the extension's version tag. According to PEP440 (link in PR description), users are encouraged to use
```py
>>> from importlib.metadata import version  
>>> version('wheel')  
'0.32.3'
```
(snippet from pydocs - link in PR description)

---
I was wondering how to use the _mkrelease_ script. I'll try this out, thanks

Reply:
*This is also a reply to https://github.com/michaeljones/breathe/pull/766#issuecomment-991959417 below.*

@2bndy5 Ah yeah, the `mkrelease` utility does rely on a POSIX-compliant environment, so in practice will need some compatibility tooling on Windows.

The way you tagged it does actually seem to be correct and the script also seems to execute correctly producing exactly the expected output. So in that case the version number isn't being detected properly by the python tooling with the new method, which is probably related to its absolute removal I mentioned here.

Effectively `mkrelease` is fairly simple and was added to automate some stuff I used to do manually every single time, the `pack` function is actually very single and can be fully found here https://github.com/michaeljones/breathe/blob/master/mkrelease#L29.

I think the problem is that a "git archive" tarball, either via CLI or download from a GitHub tag, does not contain actual git information (i.e. `.git` directory) So when the `python3 setup.py sdist bdist_wheel` is called it's called from a git-less, clean unpacked Breathe archive as-if downloaded from GitHub from the tag.

With this realisation I'm not even sure if you can truly have fully automated versioning as it has to work without the git history, meaning the current version has be stored outside of git at at least one location. Linux/BSD distributions usually package based upon the source tarball + PGP signature, not the full git source (to avoid big size and for archiving), so I don't see an easy way out of this.

Could you share thoughts about this? I'm not sure how to cleanly solve this at this point.